### ISSUE TITLE
SQLite unparsable definition: `CREATE VIRTUAL TABLE`

### ISSUE THREAD

**Issue by kgutwin** [sqlfluff/sqlfluff#6403]
### Search before asking

- [X] I searched the [issues](https://github.com/sqlfluff/sqlfluff/issues) and found no similar issues.

### What Happened

SQLite supports virtual tables (https://sqlite.org/lang_createvtab.html) as an interface to an external computation or storage engine. My use case is the FTS5 full text search module within SQLite (https://www.sqlite.org/fts5.html) which is normally defined as:

```sql
CREATE VIRTUAL TABLE email USING fts5(sender, title, body);
```

However, SQLFluff isn't able to parse this definition.

### Expected Behaviour

Parsing and linting passes successfully.

### Observed Behaviour

```
== [stdin] FAIL                                                                                                                                                
L:   1 | P:   1 |  PRS | Line 1, Position 1: Found unparsable section: 'CREATE
                       | VIRTUAL TABLE email USING fts5(se...'
WARNING: Parsing errors found and dialect is set to 'sqlite'. Have you configured your dialect correctly?
All Finished 📜 🎉!
```

### How to reproduce

```bash
echo 'CREATE VIRTUAL TABLE email USING fts5(sender, title, body)' | sqlfluff lint --dialect=sqlite -
```

### Dialect

sqlite

### Version

sqlfluff, version 3.2.5

### Configuration

Python 3.11.10

### Are you willing to work on and submit a PR to address the issue?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/sqlfluff/sqlfluff/blob/main/CODE_OF_CONDUCT.md)
